Tasks/Serzhant/Task1/Task1.py

```diff
@@ -0,0 +1,7 @@
+initial_amount = 20000
+duration = 5
+months = 12
```

Comment:
these values should be of the decimal data type, since they are money-related, -1 point for the unwise choice